politeiawww/user/cockroachdb/models.go

```diff
@@ -124,3 +124,15 @@ type CMSCodeStats struct {
 func (CMSCodeStats) TableName() string {
 	return tableCMSCodeStats
 }
+
+// EmailHistory24h represents a history of emails sent to the user with
+// a certain email address in the past 24h.
+type EmailHistory24h struct {
+	Email string `gorm:"primary_key"` // User email
```

Comment:
Can you elaborate why it is important ? We have `Email` as a primary key here just for the sake of using `Save` method as "update".

Reply:
Can you elaborate why it is important ? We have `Email` as a primary key here just for the sake of using `Save` method as "update".

Reply:
A user's email address is private data and is thus saved as part of the encrypted data blob.  This would save it unecrypted.  User lookups should be done using the uuid.